pkg/backuphelpers/backupvars.go

```diff
@@ -0,0 +1,66 @@
+package backuphelpers
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"sync"
+
+	backupv1alpha1 "github.com/openshift/api/config/v1alpha1"
+)
+
+var (
+	validBackupVars = map[string]struct{}{
+		"schedule":   {},
+		"timezone":   {},
+		"data-dir":   {},
+		"config-dir": {},
+		"backup-dir": {},
+	}
+)
+
+type BackupVar interface {
+	GetBackupVars() *BackupConfig
+}
+
+type BackupConfig struct {
+	enabled  bool
+	backupCR backupv1alpha1.EtcdBackupSpec
+	mux      sync.Mutex
+}
+
+func NewBackupConfig(backupCR backupv1alpha1.EtcdBackupSpec) *BackupConfig {
+	return &BackupConfig{
+		enabled:  false,
+		backupCR: backupCR,
+		mux:      sync.Mutex{},
+	}
+}
+
+func (b *BackupConfig) GetBackupVars() *BackupConfig {
+	b.mux.Lock()
+	defer b.mux.Unlock()
+
+	c := NewBackupConfig(b.backupCR)
+	c.enabled = b.enabled
+	return c
+}
```

Comment:
don't think we need this if we're exporting the args as string

```suggestion

```

Reply:
+1, Bon Remark :)

Reply:
done